charts/kong/templates/migrations.yaml

```diff
@@ -1,11 +1,15 @@
-{{- if (and (.Values.runMigrations) (not (eq .Values.env.database "off"))) }}
+{{- if (and (or (.Values.runMigrations) (.Values.migrations.init)) (not (eq .Values.env.database "off"))) }}
 apiVersion: batch/v1
 kind: Job
 metadata:
   name: {{ template "kong.fullname" . }}-init-migrations
   labels:
     {{- include "kong.metaLabels" . | nindent 4 }}
     app.kubernetes.io/component: init-migrations
+  annotations:
+    {{- range $key, $value := .Values.migrationAnnotations }}
+    {{ $key }}: {{ $value | quote }}
+    {{- end }}
```

Comment:
This always gets interesting. I think the original motivation is to have these annotations on the pods that are created part of the Job (spec.template.metadata.annotations). I'm not sure about this. Can you look into Istio's injection mechanism for jobs and correct this if needed?

Reply:
Nope, I just wasn't looking where I was adding it. I also forgot to change the variable name from the original `migrationAnnotations` :woman_facepalming: